benchmarks/fasta/php/bench.php

```diff
@@ -128,15 +129,16 @@ function makeRepeatFasta($s, $n) {
    'AGGCGGAGGTTGCAGTGAGCCGAGATCGCGCCACTGCACTCC' .
    'AGCCTGGGCGACAGAGCGAGACTCCGTCTCAAAAA';
 
+makeCumulative($iub);
+makeCumulative($homosapiens);
+
```

Comment:
note that that was completely inconsistent across the languages before and after this change. we could consistencify it, but probably in an extra checkin, because it requires touching all of them.